x-pack/test/functional/apps/infra/link_to.ts

```diff
@@ -12,17 +12,20 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const retry = getService('retry');
   const browser = getService('browser');
 
+  // Ensure the start/end range is kept
+  const timestamp = Date.now() - 60000;
```

Comment:
Why do we want to use a runtime-dependent timestamp for this?

Reply:
Our default range is `now-1d` to `now`. I want to make sure the timestamp always falls within that range.